app/import.go

```diff
@@ -1701,17 +1702,20 @@ func (a *App) OldImportChannel(channel *model.Channel) *model.Channel {
 	}
 }
 
-func (a *App) OldImportFile(timestamp time.Time, file io.Reader, teamId string, channelId string, userId string, fileName string) (*model.FileInfo, error) {
-	buf := bytes.NewBuffer(nil)
-	io.Copy(buf, file)
-	data := buf.Bytes()
-
-	fileInfo, err := a.DoUploadFile(timestamp, teamId, channelId, userId, fileName, data)
+func (a *App) OldImportFile(timestamp time.Time, file io.ReadCloser, teamId string, channelId string, userId string, fileName string) (*model.FileInfo, error) {
```

Comment:
Should we just make `OldImportFile` take an `io.ReadSeeker` instead of casting and handling the error?

Reply:
I also considered this method but if this change will affect slackimport.go/SlackUploadFile